marlowe-playground-client/package.json

```diff
@@ -3,14 +3,16 @@
   "version": "1.0.0",
   "scripts": {
     "webpack": "NODE_OPTIONS=\"--max-old-space-size=8192\" DEBUG=purs-loader* DEBUG_DEPTH=100 webpack --progress --bail --mode=production -p",
-    "webpack:watch": "PATH=$PATH:../releases/psc-package DEBUG=purs-loader* DEBUG_DEPTH=100 webpack --progress --display-error-details --display verbose --watch",
     "webpack:server": "webpack-dev-server --progress --inline --hot --mode=development --host 0.0.0.0 --display verbose",
+    "prewebpack:server": "npm run purs:compile",
     "webpack:server:debug": "DEBUG=purs-loader* DEBUG_DEPTH=100 webpack-dev-server --progress --inline --hot",
     "purs:compile": "spago build",
+    "prepurs:compile": "npm install && marlowe-playground-generate-purs",
     "purs:ide": "purs ide server --log-level=debug 'src/**/*.purs' 'generated/**/*.purs' 'test/**/*.purs' 'web-common/**/*.purs'",
     "test": "NODE_OPTIONS=\"--max-old-space-size=8192\" webpack --config webpack.test.config.js --mode=development && node --max-old-space-size=8192 dist/test.js",
     "docs": "spago docs",
-    "repl": "spago repl"
+    "repl": "spago repl",
+    "start": "npm install && marlowe-playground-generate-purs && npm run purs:compile && npm run webpack:server"
```

Comment:
nice

Reply:
i also added a `start` script and the generate-purescript thingy to that new dashboard project as part of this PR now